src/Contrib/StackSdks/Masa.Contrib.StackSdks.Auth/Service/UserService.cs

```diff
@@ -356,5 +356,17 @@ public async Task RemoveAsync(Guid id)
         var requestUri = "api/user";
         await _caller.DeleteAsync(requestUri, new RemoveUserModel(id));
     }
+
+    public Task BindRolesAsync(BindUserRolesModel model)
```

Comment:
line:183 `Rename parameter 'staff' to 'user' to match the interface declaration.`